index.bs

```diff
@@ -133,12 +137,35 @@ The <dfn dict-member for="XRHitTestOptionsInit">space</dfn> dictionary member sp
 
 The <dfn dict-member for="XRHitTestOptionsInit">entityTypes</dfn> dictionary member specifies array of <a enum lt="XRHitTestTrackableType">XRHitTestTrackableTypes</a> that will be used to compute results of the hit test.
 
-The <dfn dict-member for="XRHitTestOptionsInit">offsetRay</dfn> dictionary member specifies {{XRRay}} used to perform hit test. The {{XRHitTestOptionsInit/offsetRay}} is expressed in a coordinate system defined by {{XRHitTestOptionsInit/space}}.
+The <dfn dict-member for="XRHitTestOptionsInit">offsetRay</dfn> dictionary member specifies {{XRRay}} that will be used to perform hit test. The {{XRHitTestOptionsInit/offsetRay}} will be interpreted as if expressed in a coordinate system defined by {{XRHitTestOptionsInit/space}}.
 
 The {{XRHitTestOptionsInit}} dictionary has an associated <dfn for="XRHitTestOptionsInit">effective entityTypes</dfn> which is set to {{XRHitTestOptionsInit/entityTypes}} if it was provided at dictionary construction time. If the {{XRHitTestOptionsInit/entityTypes}} was not provided at construction time, the [=XRHitTestOptionsInit/effective entityTypes=] is set to an array containing single element, {{XRHitTestTrackableType/"plane"}}.
 
 The {{XRHitTestOptionsInit}} dictionary has an associated <dfn for="XRHitTestOptionsInit">effective offsetRay</dfn> which is set to {{XRHitTestOptionsInit/offsetRay}} if it was provided at dictionary construction time. If the {{XRHitTestOptionsInit/offsetRay}} was not provided at construction time, the [=XRHitTestOptionsInit/effective offsetRay=] is set to an {{XRRay}} constructed by invoking {{XRRay/XRRay()}} without any parameters.
 
+XRTransientInputHitTestOptionsInit {#transient-input-hit-test-options-dictionary}
+----------------------------------
+
+An {{XRTransientInputHitTestOptionsInit}} dictionary represents a set of configurable values that affect the behavior of the hit test for transient input that is being performed.
+
+<script type="idl">
+dictionary XRTransientInputHitTestOptionsInit {
+  required DOMString profile;
```

Comment:
I know this has the "transient" name plastered all over it, but I still suspect some people will see this and assume they can use it as an easy way to subscribe to hit tests for all inputSources with the given profile, transient or otherwise. And, honestly, maybe that's not a terrible idea? Haven't really thought through the implementation details, though.

Reply:
Current wording of the spec blocks this use case (for transient hit test results to be computed, the input source needs to be categorized as transient). I'd say we should avoid having 2 ways of doing the same thing and this variant of the API should exist _only_ to enable hit testing for transient input sources, since this is the only case which the regular hit test cannot solve.